web-platform/website/src/styles/Columns.ts

```diff
@@ -0,0 +1,66 @@
+
+import { style } from "typestyle";
+import { ScreenSize } from "./ScreenSize";
+import { Spacing } from "./Spacing";
+import { createResponsiveStyle } from "./ResponsiveStyle";
+
+export const ColumnStyles = {
```

Comment:
Thinking if we should move it to account-settings and rename the ColumnStyles to AccountSettingColumnStyles.

Reply:
Hmm, I'm not sure. We might need unique column styles for the other pages depending on what Casey creates. Let's keep it for now and if it's unnecessary, we'll move and rename it. We could also create a folder called pages in the styles dir for the pages.